typedb/common/rpc/request_builder.py

```diff
@@ -555,6 +579,11 @@ def thing_type_get_instances_req(label: Label):
     req.thing_type_get_instances_req.CopyFrom(concept_proto.ThingType.GetInstances.Req())
     return type_req(req, label)
 
+def thing_type_syntax_req(label: Label):
+    req = concept_proto.Type.Req()
+    req.thing_type_syntax_req.CopyFrom(concept_proto.ThingType.GetSyntax.Req())
```

Comment:
Typo, see protocol + rename method accordingly
```suggestion
    req.thing_type_get_syntax_req.CopyFrom(concept_proto.ThingType.GetSyntax.Req())
```